redbot/cogs/trivia/data/lists/minecraft.yml

```
@@ -0,0 +1,114 @@
+AUTHOR: ImajinDevon
+
+"The phantom was added in which update?":
+ - "1.13"
+"Which update introduced a new style combat into the game?":
+ - "1.9"
+'What edition of Minecraft is usually prefixed with the word "bug"?':
+ - "Bedrock"
+"Who created Minecraft?":
+ - "Notch"
+"Who bought Mojang in 2014?":
+ - "Microsoft"
+"Mojang was bought for (...) billion dollars:":
+ - "2.5"
+"If you name a sheep this, it will turn rainbow:":
+ - "jeb_"
+"What supercharges creepers?":
+ - "lightning"
+"What are creepers scared of?":
+ - "cats"
+"To kill an iron golem without getting hit, you must build how many blocks up?":
```

This is a reference to a specific speedrun strat, however there are many ways to kill an iron golem _without_ building up. Perhaps this question could work if the "kill" part is removed, but I still don't love that option either. I'm not convinced this is a suitable question in general.
```suggestion
"To avoid being hit by an iron golem, you must build at least how many blocks up?":
```